docs/source/releases/3.0.2.rst

```diff
@@ -0,0 +1,17 @@
+================================
+Wagtailmenus 3.0.2 release notes
+================================
+
+Minor changes
+=============
+
+* Added support for Wagtail 2.9 (no code changes necessary).
+* Added compatibility with Django Sites Framework [1]_.
+* Modify tox configuration file to ensure Django 1.11/2.1/3.0 are in dependencies.
+* Modify Travis CI configuration file to test against Django 2.1.
```

Comment:
Could we perhaps remove these two lines? I would classify these changes as part of adding support for 2.9.  Also, please add your name in brackets next to the other items to credit yourself :)